x/examples/website-wrapper-app/wrapper_app_project/template/android/app/build.gradle.handlebars

```diff
@@ -1,10 +1,10 @@
 apply plugin: 'com.android.application'
 
 android {
-    namespace "org.getoutline.pwa"
+    namespace "{{appId}}.android"
```

Comment:
Same, do we need `.android` here?

I'm ok with the `.android` in `applicationId` though.